llvm/lib/Target/Mips/MipsISelLowering.cpp

```diff
@@ -2092,6 +2095,42 @@ MachineBasicBlock *MipsTargetLowering::emitAtomicCmpSwapPartword(
   return exitMBB;
 }
 
+SDValue MipsTargetLowering::lowerREADCYCLECOUNTER(SDValue Op,
+                                                  SelectionDAG &DAG) const {
+  SmallVector<SDValue, 3> Results;
+  SDLoc DL(Op);
+  unsigned RdhwrOpc, DestReg;
+
+  if (Subtarget.hasMips64()) {
+    RdhwrOpc = Mips::RDHWR64;
+    DestReg = Mips::V1_64;
```

Comment:
Why can V1 can hardcoded here?
If the IR is quite long, will it overwrite some previous value?